draft-thoji-scone-trone-protocol.md

```diff
@@ -372,6 +372,63 @@ TRONE packets could be stripped from datagrams in the network, which cannot be
 reliably detected.  This could result in a sender falsely believing that no
 network element applied a rate limit signal.
 
+## Interactions with congestion control
+
+TRONE and congestion control both provide the application with estimates
+of a path capacity. They are complementary. Congestion control algorithms
+are typically designed to quickly detect and react to congestion, i.e., to
+the "minimum" capacity of a path. TRONE informs the endpoint
+of the maximum capacity of a path.
+
+Consider for example a path in which the bottleneck router implements Early
+Congestion Notification as specified in the L4S architecture {{?RFC9330}}.
+If the path capacity diminishes, queues will build up and the router
+will immediately start increasing the rate at which packets are marked
+as "Congestion Experienced". The receiving endpoint will notice these marks,
+and inform its peer. The incoming congestion will be detected within
+1 round trip time (RTT). This scenario will play out whatever the reason
+for the change in capacity, whether due to increased competition between
+multiple applications or, for example, to a change in capacity of a wireless
+channel.
```

Comment:
I think text up to this paragraph if fine.

But for the remainder, I'm not sure if it'd be a good idea to imply TRONE as a signal that supplements congestion signals that indicate that more bandwidth might be available.

I wonder if it would be better to explain the difference of the time scales congestion signals and TRONE signals act upon.

The time scale of congestion signals is in the unit of RTT, while I'd presume TRONE signals to indicate average b/w available across ~10 seconds; see https://github.com/ietf-wg-scone/trone/issues/20. For video playback using chunked files, the two signals would be handled differently; congestion control applies to the transport, while the TRONE signal would be used for choosing the chunks with the appropriate bitrate.

Reply:
Agreed. The following paragraphs are cut in the next commit.